api/docs/bt.dox

```diff
@@ -89,6 +89,19 @@ a primary contributor to DynamoRIO's efficiency.
 The instruction representation includes all of the operands, whether
 implicit or explicit, and the condition code effects of each instruction.
 This allows for analysis of liveness of registers and condition codes.
+The operands are split into sources and destinations.
+
+A memory reference is treated as one operand even when it uses
+registers to compute its address: those constituent registers are not
+listed as their own separate source operands (unless they are read for
+other reasons such as updating the index register).  This means that a
+store to memory will have that store as a destination operand without
+listing the store's addressing mode registers as source operands in
+their own right.  Tools interested in all registers inside such
+operands can use interfaces like instr_reads_from_reg() to query (it
```

Comment:
When I looked at `instr_reads_from_reg()` it didn't seem like a good way to "query all the registers inside an operand".
My understanding is that `instr_reads_from_reg(instr_t *instr, reg_id_t reg, ...)` will simply tell you if `reg` is used inside any operand of `instr`, but to "query all the registers" as suggested, one would have to iterate over all registers and call `instr_reads_from_reg()` for every one of them.
If we want to suggest a way to query registers from operands, I think it makes more sense to mention: `int opnd_get_num_regs_used(opnd_t opnd)` and `reg_id_t opnd_get_reg_used(opnd_t opnd, int index)`.
Although now we'd have to at least explain what opnd_t is (which is fine btw, we could just mention it in the previous paragraph, something like: "The instruction representation includes all of the operands (which opnd_t represents), whether ... ").

All of this, assuming my understanding of `instr_reads_from_reg()` is correct.